cc3d/core/GraphicsUtils/CC3DPyGraphicsFrame.py

```diff
@@ -14,7 +14,12 @@
 from typing import Any, Callable, Dict, List
 import warnings
 from vtkmodules.vtkInteractionStyle import vtkInteractorStyleTrackballCamera
-from vtkmodules.vtkIOExportPython import vtkGL2PSExporter
+try:
+    # vtk 8
+    from vtkmodules.vtkIOExportPython import vtkGL2PSExporter
+except ModuleNotFoundError:
+    # vtk 9
```

Comment:
This change doesn't seem to be in the current windows conda package

Reply:
It might be , I can regenerate this package